From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sun, 2 Jun 2019 19:47:07 -0400
Subject: barf: fix broken shellscript

This is a combination of 8c3ba6a5f73ae64aa5171252f54c15d65c9930db and
part of 716f4b712724c6698469563e531dea3667507ceb from upstream.

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
---
 programs/barf/barf.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/programs/barf/barf.in b/programs/barf/barf.in
index fce0599..491b188 100755
--- a/programs/barf/barf.in
+++ b/programs/barf/barf.in
@@ -170,6 +170,8 @@ if test -r /proc/net/ipsec_tncfg
 then
 	cat /proc/net/ipsec_tncfg
 fi
+if test -r /proc/sys/net/core/xfrm_acq_expires
+then
 _________________________ ip-xfrm-state
 	ip xfrm state
 _________________________ ip-xfrm-policy
